CollaboraOnline/online
# private/kendy/hcastro rename

- rename loleaflet.html -> cool.html
- rename loleaflet-help.html -> cool-help.html
- loleaflet: makefile: change vars LOLEAFLET -> COOL
- initial rename loleaflet -> browser
- l10n: rename loleaflet -> browser
- update .gitignore, rename loleaflet -> browser
- wsd: update protocol.txt, rename loleaflet -> browser
- wsd: update README, rename loleaflet -> browser
- update .gitpod-files, rename loleaflet -> browser
- browser: rename loleaflet-api -> cool-api
- browser: update cool-api.html
- browser: update CSS class names
- browser: update npm-shrinkwrap.json
- browser: rename trace events file name
- browser: src: replace CSS class names
- browser: po: update loleaflet -> cool
- browser: update README
- cypress: update CSS class name
- cypress: update .eslintrc
- cypress: update README file
- cypress: update Makefile.am
- test: rename loleaflet -> browser
- test: rename loleaflet -> cool
- docker: update comment
- cypress: rename loleaflet -> browser
- apache2: update loolwsd.conf file
- nginx: update loolwsd.conf file
- android: rename loleaflet -> browser
- android: rename loleaflet -> cool
- makefile: rename loleaflet -> browser
- update loolwsd.spec.in
- update README file
- scripts: rename loleaflet -> browser
- browser: update package.json
- browser: test: update load.js file
- browser: update Makefile.am
- browser: update JS files
- browser: admin: rename loleaflet -> browser
- browser: html: update cool.html.m4
- browser: html: update more files
- browser: rename makefile vars
- browser: update loolwsd.xml.in file
- ios: rename loleaflet -> browser
- gtk: rename loleaflet -> browser
- browser: update config files
- wsd: update comments
- wsd: rename loleaflet -> browser
- wsd: update define variable
- wsd: update trace prefix and comments
- kit: update comments
- configure: update file
- rename var LOLEAFLET_LOGGING to
- wsd: more variable renames
- browser: rename api from loleaflet -> cool
- browser: rename file loleaflet.css
- update missing replaces after rebase
- codeql: ignore paths related to w2ui library
- browser: move location debug.html
- codeql: ignore old "loleaflet" directory
- Remove even the loleaflet/ dir if it is around
- debug.html: Should point to cool.html too
